mantidproject/mantid
# Removed the "Test the Curve Fit widget" interface

This interface was a test harness for a developer and is not used anymore.
Immediate removal was agreed by the dev team leads.


**To test:**

1. Start Mantidplot verify there is no "test the ...." interface under Interfaces->DynamicPDF
1. Verify the code has been removed
1. There were no help pages

Fixes #21138 .

**Release Notes** 

docs/source/release/.../diffraction.rst

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
